src/redisearch_rs/inverted_index/src/raw_doc_ids_only.rs

```diff
@@ -22,6 +22,7 @@ pub struct RawDocIdsOnly;
 
 impl Encoder for RawDocIdsOnly {
     type Delta = u32;
+    const RECOMMENDED_BLOCK_ENTRIES: usize = 1000;
```

Comment:
This encoder should also override the `delta_base` method to use the first doc id of the block instead.

https://github.com/RediSearch/RediSearch/blob/768dd32a7d992fa940eff551677cdf00916fe8cc/src/inverted_index/inverted_index.c#L661-L665

That will allow us to remove this todo

https://github.com/RediSearch/RediSearch/blob/768dd32a7d992fa940eff551677cdf00916fe8cc/src/redisearch_rs/inverted_index/src/lib.rs#L154-L155

Reply:
I added a commit doing that.